.travis.yml

```diff
@@ -30,13 +36,15 @@ install:
     - pip install --upgrade pyopenssl ndg-httpsclient pyasn1
     - pip install coveralls flake8
 
+    # Install tox
+    - pip install tox
```

Comment:
If you run locally:
```python
cd pylint-odoo
tox
```
You will be tested 2 virtual environments of pylint-odoo
 - pylint1.4
 - pylint2.0

tox is the most library used in python to multi-environment cases.